ubiconfig/utils/config_schema.json

```diff
@@ -8,49 +8,62 @@
                 "input": {"type": "string"},
                 "output": {"type": "string"}
                 },
+            "additionalProperties": false,
             "required": ["input", "output"]
             },
         "module": {
             "type": "object",
             "properties": {
                 "name": {"type": "string"},
-                "stream": {"type": "number"},
+                "stream": {"type": ["string", "number"]},
```

Comment:
Shouldn't it be one or the other?  Or alternatively, can the Python classes always return this as a string regardless of what was in the config file?

I guess what this is about is that if someone writes:

  stream: 10

or:

  stream: v10

The first will "helpfully" be loaded as a number (could be written as "10" instead to avoid this).

But in truth, the stream field is really a string.  I believe either it should be enforced as such here, or else you can accept loading either a string or number but then the Python module class should always return a string.

Reply:
it can either be numbers, but in some cases, I also see it's 'rhel8', which crashed when it's set to number, so I make it either string or number. 
I'm not sure if python classes can always return this as a string or not, I can check that.

Reply:
Hmm, misunderstood you, I was thinking make yaml load numbers as strings no matter what's in config file, and I did find some ways.  
But yeah, the Module class has already made it return a string, so I guess we can spare that.